src/schema/feeds.ts

```diff
@@ -1371,9 +1373,21 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
     feedByIds: feedResolver(
       (ctx, { postIds }: FeedArgs & { postIds: string[] }, builder, alias) =>
         fixedIdsFeedBuilder(ctx, postIds, builder, alias),
+
       feedPageGenerator,
       applyFeedPaging,
-      {},
+      {
+        fetchQueryParams: async (ctx): Promise<void> => {
+          const isTeamMember = await ctx.con
+            .getRepository(Feature)
+            .findOneByOrFail({ feature: FeatureType.Team, userId: ctx.userId });
+          if (!isTeamMember || isTeamMember.value !== 1) {
```

Comment:
Nice! Can you add a test as well? 🙏

Reply:
Wanted to avoid it as it's only internal nonsense but yeah 🤣  (just laziness)

Reply:
@capJavert Put my laziness aside and added test cases let me know if we good to go?